source/connect-to-govwifi/device-iphone-or-ipad.html.erb

```diff
@@ -22,14 +21,14 @@ description: Follow these instructions to connect your iPhone or iPad to GovWifi
           </li>
           <li><%= partial 'shared/copy/password_instructions' %></li>
           <li>
-            <p class="govuk-body">You'll be asked to confirm that the network presents the correct certificate. This protects you against fake networks which may harm your privacy.</p>
-            <p class="govuk-body">The correct certificate is called <strong>wifi.service.gov.uk</strong> and is issued by <strong>GeoTrust RSA CA 2018</strong>.</p>
+            <p class="govuk-body">You'll be shown the 'certificate' and asked to trust it.</p>
+            <p class="govuk-body">Your device should show that the certificate is called <strong>wifi.service.gov.uk</strong> and is issued by <strong>GeoTrust RSA CA 2018</strong>.</p>
+            <p class="govuk-body">It might also say ‘not trusted’, which just means your device does not recognise the network yet.</p>
```

Comment:
very small thing but the capitalisation is different from the old version - is it lower case 'n' in 'not trusted'?

Reply:
Ah no it should be a capital - ta